### PR TITLE
add design doc

### PR DESCRIPTION
**Pull request by kaka89**
add design doc: mainly add event design.

### CODE REVIEW

File: docs/design.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  Integer quantity;
+```
+- Money:
+```java
+  String currencyCode;
+  String centAmount;
+```
+
+### 4.2 Consumer Of Order Payed Event
+When an order is payed by `payment` service, it will produce an event, the order consume this event, and change order's status.
```
Comment (davis999) [inline docs/design.md]: order service consumes this event.

File: docs/design.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+`order-payment-payed`
+#### 4.2.2 Model
+
+#### 4.2.3 Workflow
+1. get order id, payment id, pay result and result message from event message
+2. validate if the order with order id
+3. change order's status and save the pay result
+
+### 4.3 Consumer Of Order Reserved Event
+When an order is reserved by inventory service, it will produce an event, the order consume this event, and change order's status.
```
Comment (davis999) [inline docs/design.md]: order service consumes this event.

File: docs/design.md (hunk 1 above)
Comment (kaka89) [reply docs/design.md]: done.

File: docs/design.md (hunk 2 above)
Comment (kaka89) [reply docs/design.md]: done.

**Pull request merged**